app/views/controllers/application/sidebar/_user.html.erb

```diff
@@ -2,18 +2,13 @@
   [
     tag.i("", class: "glyphicon glyphicon-user"),
     tag.span(h(@user.login), class: "ml-2"),
-    tag.span(class: "pull-right") do
-      link_to(:app_logout.t, account_logout_path, id: "nav_user_logout_link")
+    tag.span(class: "pull-left") do
+      button_to(:app_logout.t,
+                account_logout_path,
+                { class: "btn", id: "nav_user_logout_link" })
     end
```

Comment:
Should this be centered or otherwise moved?

Reply:
This one currently flushes right, but it doesn't really work with long usernames > "chaelthomas".
I propose knocking it down to the next line... committed that change, let me know if that works.

Reply:
(Trying to avoid the situation where it currently smushes the username on top of the word "logout")